### ISSUE TITLE
memcache sasl  auth

### ISSUE THREAD

**Issue by peterpaule** [openresty/memc-nginx-module#19]
Can this module support   sasl  username and password?  If it can, how? help

**Comment by agentzh** [comment]
@peterpaule No, it can't. I think this would require the support of the binary wire protocol of memcached. This module currently supports the text wire protocol only. If you would like to add the binary support, patches welcome :)

**Comment by peterpaule** [comment]
Thanks, I think it's usefull for adding the feature to this module;
